backend/pick-git/src/test/java/com/woowacourse/pickgit/post/application/PostServiceTest.java

```diff
@@ -0,0 +1,48 @@
+package com.woowacourse.pickgit.post.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.woowacourse.pickgit.post.application.dto.TokenDto;
+import com.woowacourse.pickgit.post.domain.PlatformExtractor;
+import com.woowacourse.pickgit.post.infrastructure.dto.RepositoryResponse;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class PostServiceTest {
+
+    @InjectMocks
+    PostService postService;
+
+    @Mock
+    PlatformExtractor platformExtractor;
+
+    @DisplayName("Repository 목록을 가져온다.")
+    @Test
+    void getRepositories_AuthenticatedUser_Success() {
+        // given
+        TokenDto tokenDto = new TokenDto("pickgit");
+        List<RepositoryResponse> repositories = List.of(
+            new RepositoryResponse("pick"),
+            new RepositoryResponse("git")
+        );
+
+        // when
+        given(platformExtractor.getRepositories(tokenDto.getAccessToken()))
+            .willReturn(repositories);
+
+        // then
+        assertThat(platformExtractor.getRepositories(tokenDto.getAccessToken()))
+            .containsAll(repositories);
+        verify(platformExtractor, times(1))
+            .getRepositories(tokenDto.getAccessToken());
+    }
```

Comment:
```suggestion
    void getRepositories_AuthenticatedUser_Success() {
        // given
        TokenDto tokenDto = new TokenDto("pickgit");
        List<RepositoryResponse> repositories = List.of(
            new RepositoryResponse("pick"),
            new RepositoryResponse("git")
        );
        given(platformExtractor.getRepositories(tokenDto.getAccessToken()))
            .willReturn(repositories);

        //when
        List<RepositoryResponse> repositories1 = platformExtractor
            .getRepositories(tokenDto.getAccessToken());

        //then
        assertThat(repositories1)
            .containsAll(repositories);
        verify(platformExtractor, times(1))
            .getRepositories(tokenDto.getAccessToken());
    }
```

given when then 은 위와같이 나뉩니다!!
아래 테스트 코드도 마찬가지에요!